patches/server/0009-Timings-v2.patch

```diff
@@ -473,7 +473,7 @@ index 0000000000000000000000000000000000000000..f27fadc15cb7f5c782e45885ec6a5a69
 +        String response = null;
 +        String timingsURL = null;
 +        try {
-+            HttpURLConnection con = (HttpURLConnection) new URL("http://timings.aikar.co/post").openConnection();
```

Comment:
Feels like the endpoint too should be configurable, really...

Reply:
I don't think that should be exposed as lots of the logic is dependent on what the Timings Viewer imposes and the endpoint name is part of that. Yes one could theoretically proxy that or rename it in a fork but you could also just do the opposite to be compatible. Exposing this could lead to a lot more user errors imo than only exposing the base-url.

Reply:
Fair enough.